syntax = "proto3";

message BitVector {
    int64 len = 1;
    bytes data = 2;
}
